### ISSUE TITLE
Editing API key results in 404 error

### ISSUE THREAD

**Issue by NiceGuyIT** [amidaware/tacticalrmm#1369]
**Server Info (please complete the following information):**
 - OS: Ubuntu 20.04.4 LTS (Focal Fossa)
 - Browser: Firefox 107.0 (64-bit)
 - RMM Version (as shown in top left of web UI): v0.15.4

**Installation Method:**
  - [X] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): N/A
- Agent OS: N/A

**Describe the bug**
Editing an API key returns an error.

**To Reproduce**
1. Go to Settings > Global Settings > API Keys
2. Add a new API key
3. Right click > Edit
4. Change the date and click Submit.
5. An error is displayed across the top. The returned document is HTML.

```html
<!doctype html>
<html lang="en">
<head>
  <title>Not Found</title>
</head>
<body>
  <h1>Not Found</h1><p>The requested resource was not found on this server.</p>
</body>
</html>
```

**Expected behavior**
I expect the API key to be updated.

**Screenshots**
N/A

**Additional context**
The frontend submits a POST to `/accounts/apikeys/[object%20Object]/`. I suspect `[object%20Object]` is incorrect because creating a new API key submits a POST to `/accounts/apikeys/`.

As a side note, shouldn't the backend return a JSON instead of HTML for 404 errors?

**Comment by wh1te909** [comment]
thanks, fixed in https://github.com/amidaware/tacticalrmm-web/commit/8f34f76a1dc2140c46391c3938f33de20bbf1675
no the default behavior is to return html 404, not json. that would require a custom override

**Comment by wh1te909** [comment]
fixed in 0.15.6
